utils.js

```diff
@@ -0,0 +1,19 @@
+async function retry(fn, retries = 1) {
+    let currentTry = 0;
+    let lastError;
+    while (currentTry < retries) {
+        try {
+            const result = await fn();
+            return result;
+        } catch (err) {
+            console.error(err);
+            currentTry++;
```

Comment:
maybe it will be a good idea to add a little bit time between retries?

Reply:
Main reason I have added this code is timeouts throwing by report portal (default is 30sec), so I'm not sure if additional wait is needed. Will see how integration testing goes.
Anyway I'm not gonna to merge it without total refactoring formatter lib

Reply:
got it